src/__tests__/test.ts

```diff
@@ -1,7 +1,24 @@
-import { myModule } from '..';
+import { ndParse } from '..';
 
-describe('test myModule', () => {
-  it('should return 42', () => {
-    expect(myModule()).toBe(42);
+const simpleExample = `
+meta.first,1
+meta.second,,2
+temp, volt, curr
+1,1,1
+2,2,2
+3,3,3
+`;
+
+test('simpleExample', () => {
+  expect(ndParse(simpleExample)).toStrictEqual({
+    data: {
+      temp: { data: [1, 2, 3], label: 'temp' },
+      volt: { data: [1, 2, 3], label: 'volt' },
+      curr: { data: [1, 2, 3], label: 'curr' },
+    },
+    meta: {
+      'meta.first': '1',
```

Comment:
May have to think about an 'autoTyping' option

Reply:
I think that this kind of decisions should be taken in the library that will use it